apps/examples/testcase/le_tc/kernel/Kconfig

```diff
@@ -197,6 +198,11 @@ config TC_KERNEL_MEMORY_SAFETY
 	default n
 	depends on !DISABLE_MQUEUE
 
+config TC_KERNEL_IRQ
+	bool "irq"
+	default n
+	depends on S5J_MCT
```

Comment:
why this dependency? is it related to specific board

Reply:
corrected.